client/src/entities/min.entities.ts

```diff
@@ -0,0 +1,17 @@
+export interface GetPricePairs {
```

Comment:
I think we can start standardizing fetch request result using DTO

`GetPricePairsDto`

What do you think?